book/src/pil/declarations.md

```diff
@@ -0,0 +1,27 @@
+# Declarations
+
+Powdr-pil allows the same syntax to declare various kinds of symbols. This includes
+constants, fixed columns, witness columns and even macros. It deduces the symbol kind
+by its type and the way the symbol is used.
+
```

Comment:
Maybe worth mentioning here that the default type is the given field element?

Reply:
Actually the way it is implemented now is more like python. Maybe better to not mention anything until we have a proper type checker?